aws/kustomization.yaml

```diff
@@ -16,5 +16,6 @@ resources:
 - v14.0.0
 - v14.1.1
 - v14.2.0
+- v20.0.0
```

Comment:
should it be named something like `v20.0.0-alpha1`?

Reply:
We could do that. Right now I used this name since it is the one used in the `kyverno policies` https://github.com/giantswarm/kyverno-policies